data/countries.csv

```diff
@@ -227,7 +227,7 @@ Tokelau,TK,eng;smo;tkl,🇹🇰
 Tonga,TO,eng;ton,🇹🇴
 Trinidad and Tobago,TT,eng,🇹🇹
 Tunisia,TN,ara,🇹🇳
-Turkey,TR,tur,🇹🇷
+Türkiye,TR,tur,🇹🇷

```

Comment:
```suggestion
Turkiye,TR,tur,🇹🇷
```

We use only Latin letters in country names to make it easier to sort them alphabetically.